src/pages/index.js

```diff
@@ -94,7 +203,15 @@ const editFormValidator = new FormValidator(
 
 const addFormValidator = new FormValidator(validationSettings, addFormElement);
 
+const avatarFormValidator = new FormValidator(
+  validationSettings,
+  AvatarEditForm
+);
+
 editFormValidator.enableValidation();
 addFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
-cardList.renderItems();
+api.getInitialCards().then((cardsData) => {
+  cardList.renderItems(cardsData);
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**